### DISCUSSION TITLE
How to output total water storage  (TWS)

### DISCUSSION THREAD

**Discussion by walter-git** [iiasa/CWatM#32] [Q&A] [answered]
Hi everyone,
Hope you all are doing well.
I want to compute the total storage water (TWS) and compare it with GRACE.
But I don't know how to output the variable TWS in CWATM.
Please help me this regard.
Thank you very much.
Meishui Li

**Comment by walter-git** [comment]
Thank you, I have found the answer.

**Comment by mikhailsmilovic** [comment, marked as answer]
The variable _tws_ outputs total water storage.

The separate storage components as outputs include:
_storGroundwater, smalllakeStorage, lakeResStorage, channelStorage, totalSto_

totalSto can be output as its separate components, namely _SnowCover, sum_interceptStor_, and _sum_soil_

**Comment by walter-git** [comment]
Thank you very much!
